rts/Rendering/Env/Particles/ProjectileDrawer.cpp

```diff
@@ -872,6 +929,9 @@ void CProjectileDrawer::DrawShadowPassTransparent()
 	// 1) Render opaque objects into depth stencil texture from light's point of view - done elsewhere
 
 	// draw the model-less projectiles
+	if (DRAW_ONLY_VISIBLE_PARTICLE) {
+		DrawProjectilesSetShadow(visibleParticles);
+	} else
 	DrawProjectilesSetShadow(modellessProjectiles);
```

Comment:
alignment should be fixed. Also inconsistent formating: previous condition is encased in brackets and this one isn't. They either both should be or both should not. Ideally formating to should be kept consistent with rest of the file.